cluster/gce/config-common.sh

```diff
@@ -84,6 +84,20 @@ function get-node-ip-range {
   echo "${suggested_range}"
 }
 
+function get-cluster-ip-range {
```

Comment:
Yes, we can set it there as well (and I'll follow up this change with that). However we need them here too because our kube-up script here can be used independent of test-infra.